docs/clients/s3.md

```diff
@@ -117,6 +117,18 @@ use AsyncAws\S3\S3Client;
 $s3 = new S3Client(['pathStyleEndpoint' => true]);
 ```
 
+## Chunked body
+
+When sending data to AWS endpoints, AsyncAws split the content in multiple
+chunks. To change this behavior, set `sendChunkedBody` parameter to `false` when
```

Comment:
Maybe explain why we are splitting it into chunks. 

Maybe also mention why you would like to disable it.

Reply:
doc updated